main.cpp

```diff
@@ -18,13 +18,24 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+private: Rewriter &rewriter;
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& rewriter_):rewriter(rewriter_) {
         // Your code goes here
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
         // Your code goes here
+        const auto *StyleCastExpr=Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `StyleCastExpr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
add this  if(StyleCastExpr==nullptr) { return; }